RestSharpClient/ApiClient/ApiMethodParams.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace RestSharpClient.ApiClient
+{
+	public class ApiMethodParams
+	{
+        public string? Uri { get; set; }
+		public Dictionary<string, string>? QueryParams { get; set; }
+        public Dictionary<string, string>? Headers { get; set; }
+        public Dictionary<string, string>? UrlSegments { get; set; }
+        public Object? Payload { get; set; }
```

Comment:
please use strongly typed data, avoid Object. 
You can use Generics here if you want

Reply:
Fixed this object with generics and fixed all the affected areas. Thanks!